src/color-library.ts

```diff
@@ -28,6 +28,56 @@ export function getInactiveForegroundColorHex(backgroundColor = '') {
   return formatHex(tinycolor.mix(foreground, background, 25));
 }
 
+export function getReadableAccentColorHex(
+  backgroundColor = '',
+  ratio: ReadabilityRatios = ReadabilityRatios.Text
```

Comment:
Can we remove the type, since it has a default value which is of this type and it can be inferred?

Reply:
Yes, same response as my previous comment 😄